story.py

```diff
@@ -15,23 +16,46 @@ def read_tsv():
 	  story = row[1]
 	  prompt = row[2]
 	  next_choices = row[3]
-
 	  for choice in prev_choices.split(','):
-	  	# clean up/standardize tsv data
-		if '0' not in choice:
+	  	if '0' not in choice:
 	  		choice = '0.' + choice
```

Comment:
should do the same for next_choices and factor out into a separate function while you're at it.